### ISSUE TITLE
[TASK] Fix Lighthouse failure - Summary

### ISSUE THREAD

**Issue by MagdalenaLarge** [ONSdigital/design-team#219]
Alessio addressed Lighthouse failure in the ticket https://github.com/ONSdigital/design-system/issues/3350.

As an outcome of the above ticket, identified a need to address the warnings for all below components. It relates to accessibility.

List of components

1 result(s) for http://localhost:39203/components/summary/example-summary.html
1 result(s) for http://localhost:39203/components/summary/example-summary-no-action.html
1 result(s) for http://localhost:39203/components/summary/example-summary-multiple.html
1 result(s) for http://localhost:39203/components/summary/example-summary-hub.html
1 result(s) for http://localhost:39203/components/summary/example-summary-hub-minimal.html
1 result(s) for http://localhost:39203/components/summary/example-summary-household.html
1 result(s) for http://localhost:39203/components/summary/example-summary-grouped.html
1 result(s) for http://localhost:39203/components/summary/example-summary-grouped-with-errors.html
1 result(s) for http://localhost:39203/components/summary/example-summary-grouped-total.html
1 result(s) for http://localhost:39203/components/summary/example-summary-card-grouped.html

**Action**
Use yarn start and then move onto the above components in order. Use the lighthouse test from the google chrome dev tools. Run the test on the component and only look at the accessibility score. Fix the issues highlighted under the accessibility section. Run the test again and confirm that they are now at 100. Move on to the next component.

**Comment by rmccar** [comment]
This one seems to be an issue with the elements we have added inside `<dl>` elements, we have our `<dt>` and `<dd>` elements within two divs and then inside a `<dl>`. It is only valid to have one level of divs between them and the `<dl>`.
